torch/testing/_internal/common_modules.py

```diff
@@ -2524,6 +2526,8 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
                ),
     ModuleInfo(torch.nn.AdaptiveAvgPool2d,
                gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
+               # Fails on backward check if output size is 1x1
+               gradcheck_memformat=False,
```

Comment:
Appreciate the detailed comments for the reasons why each of these are marked False! Instead of `gradcheck_memformat` could we xfail these? That way if the failing tests are fixed we will have signal in CI